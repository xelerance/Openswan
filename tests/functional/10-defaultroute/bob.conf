# /etc/ipsec.conf - strongSwan IPsec configuration file

version 2

config setup
         nat_traversal=yes
         virtual_private=%v4:10.0.0.0/8,%v4:192.168.0.0/16,%v4:172.16.0.0/12,%v4:25.0.0.0/8,%v6:fd00::/8,%v6:fe80::/10

conn %default
        ikelifetime=60m
        salifetime=20m
        rekeymargin=3m
        keyingtries=1

conn nat-t
        left=%defaultroute
        leftcert=bobCert.pem
        leftid=bob@strongswan.org
        right=%any
        rightsubnet=10.1.0.0/16
        auto=add
