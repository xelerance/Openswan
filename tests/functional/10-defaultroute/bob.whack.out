d9 d9f8         # tag(55800)
   da 4f50534e  # tag(1330664270)
      43        # bytes(3)
         424f52 # "BOR"
a2                                      # map(2)
   04                                   # unsigned(4)
   aa                                   # map(10)
      01                                # unsigned(1)
      65                                # text(5)
         6e61742d74                     # "nat-t"
      03                                # unsigned(3)
      a6                                # map(6)
         05                             # unsigned(5)
         72                             # text(18)
            626f62407374726f6e677377616e2e6f7267 # "bob@strongswan.org"
         06                             # unsigned(6)
         6b                             # text(11)
            626f62436572742e70656d      # "bobCert.pem"
         0f                             # unsigned(15)
         01                             # unsigned(1)
         10                             # unsigned(16)
         03                             # unsigned(3)
         14                             # unsigned(20)
         19 01f4                        # unsigned(500)
         18 8f                          # unsigned(143)
         03                             # unsigned(3)
      04                                # unsigned(4)
      a4                                # map(4)
         0f                             # unsigned(15)
         02                             # unsigned(2)
         11                             # unsigned(17)
         01                             # unsigned(1)
         0e                             # unsigned(14)
         d8 34                          # tag(52)
            82                          # array(2)
               10                       # unsigned(16)
               42                       # bytes(2)
                  0a01                  # "\n\x01"
         14                             # unsigned(20)
         19 01f4                        # unsigned(500)
      18 7f                             # unsigned(127)
      1b 0000000102000066               # unsigned(4328521830)
      18 92                             # unsigned(146)
      19 0e10                           # unsigned(3600)
      18 93                             # unsigned(147)
      19 04b0                           # unsigned(1200)
      18 94                             # unsigned(148)
      18 b4                             # unsigned(180)
      18 95                             # unsigned(149)
      18 64                             # unsigned(100)
      18 96                             # unsigned(150)
      01                                # unsigned(1)
      12                                # unsigned(18)
      00                                # unsigned(0)
   0b                                   # unsigned(11)
   65                                   # text(5)
      6e61742d74                        # "nat-t"
